app/institutions/dashboard/-components/departments-panel/component.ts

```diff
@@ -1,13 +1,18 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
 import { ChartData, ChartOptions, Shape } from 'ember-cli-chart';
+import Intl from 'ember-intl/services/intl';
 import { Department } from 'ember-osf-web/models/institution';
 import InstitutionDepartmentsModel from 'ember-osf-web/models/institution-department';
 
 export default class DepartmentsPanel extends Component {
-    departments!: InstitutionDepartmentsModel[];
+    @service intl!: Intl;
 
-    chartHoverIndex: number = -1;
+    topDepartments!: InstitutionDepartmentsModel[];
+    totalUsers!: number;
+
+    chartHoverIndex: number = 0;
```

Comment:
with `chartHoverIndex` initialized to 0 as opposed to -1, shouldn't `chartOptions.onHover` callback be updated somehow? Also shouldn't we bind `chartOptions.onHover` to `DepartmentsPanel` `this` context, seeing as we are referencing `this.set` and `this.chartHoverIndex`.